packages/util-logger/package.json

```diff
@@ -19,21 +19,20 @@
   },
   "devDependencies": {
     "jest": "25.3.0",
-    "jest-esm-transformer": "1.0.0",
-    "microbundle": "0.12.0-next.8"
+    "jest-esm-transformer": "1.0.0"
   },
-  "source": "src/util-logger.js",
+  "build": "src/util-logger.js",
   "main": "dist/util-logger.js",
   "module": "dist/util-logger.module.js",
   "unpkg": "dist/util-logger.umd.js",
   "scripts": {
     "prepublishOnly": "npm test && npm run build",
     "test": "jest",
-    "build": "microbundle --no-compress",
+    "build": "fcl-bundle",
     "test:watch": "jest --watch",
-    "start": "microbundle watch --no-compress"
+    "start": "fcl-bundle --watch"
   },
   "dependencies": {
-    "@onflow/sdk": "^1.1.0-alpha.2"
```

Comment:
@onflow/sdk was not used within @onflow/util-logger and was actually a circular dependency which I just removed in this PR since lerna was throwing warnings about it

Reply:
`@onflow/util-logger` imports from `@onflow/config` so well need to add it as a dependency here